testsuite/MDAnalysisTests/analysis/test_encore.py

```diff
@@ -148,7 +150,8 @@ def test_rmsd_matrix_with_superimposition(self, ens1):
 
         for i,rmsd in enumerate(reference.rmsd):
             assert_almost_equal(conf_dist_matrix[0,i], rmsd[2], decimal=3,
-                                err_msg = "calculated RMSD values differ from the reference implementation")
+                                err_msg="calculated RMSD values differ from \
+                                    the reference implementation")
```

Comment:
As mentioned in https://github.com/MDAnalysis/mdanalysis/pull/3631#discussion_r851651529 using `\` for continuation is highly discouraged by PEP8.

Please define the error message on the line before the `assert_almost_equal` (using implicit continuation) and then pass it here as a variable.